CONTRIBUTING.md

```diff
@@ -81,7 +81,7 @@ Głowną różnicą między Issue, a Pull Request jest to, że do stworzenia Pul
 * Jeżeli wyskoczy komunikat "Sorry, it looks like your fork is outdated!", naciśnij "Update your fork".
 * Wprowadź zmiany.
 * W prawym górnym rogu widoku pliku znajduje się zielony przycisk "Commit changes...". Jeżeli skończyłeś edytować, naciśnij go.
-* W oknie "Commit message" dajemy opis tak zwanego commita. Commity to zmiany, między którymi można przechodzić. Commitów w PR może być wiele. Napisz tam prostą, klarnowną wiadomość czego dotyczą twoje zmiany. Przykładowo może to być "Zaktualizuj Art. 2. rok podatkowy".
+* W oknie "Commit message" dajemy opis tak zwanego commita. Commity to zmiany, między którymi można przechodzić. Commitów w PR może być wiele. Napisz tam prostą, klarnowną wiadomość czego dotyczą twoje zmiany. Przykładowo może to być "Zaktualizuj Art. 2. rok podatkowy". W miarę możliwości pisz commity po polsku.
```

Comment:
> W miarę możliwości pisz commity po polsku.
Przepraszam, ale to zdanie nie ma sensu co najmniej z dwóch powodów:
1. w miarę możliwości jest mocno subiektywne
2. commity po polsku.... no to po polsku czy nie? Jeśli chcemy po polsku to używajmy "zatwierdzenie" ; "zatwierdzenie zmian"

Ogólnie nie widzę sensu w dyskusjach "Słuchajcie piszmy po polsku" "Lepiej po angielsku" .... dajmy sobie z tym spokój i piszmy naturalnym językiem społeczności, którym w obecnych czasach jest mieszanka polskiego i angielskiego.